doc/grass_database.html

```diff
@@ -222,18 +222,26 @@ <h3>Starting GRASS GIS using command line</h3>
 in a given Mapset with only command line interface:
 
 <div class="code"><pre>
-grass79 --text ~/grassdata/mylocation/mymapset
+# Linux, Mac, *BSD, ...:
+grass80 --text ~/grassdata/nc_spm_08_grass7/user1
+
+# Windows
+grass80 --text D:\grassdata\nc_spm_08_grass7\user1
 </pre></div>
```

Comment:
Do you think that someone who has an issue figuring out the need for a Windows path from the unix example (as before) will parse through the `#` comments in the example? Either delete the Windows example or make it a separate paragraph.

Reply:
What do you mean? In my experience, students want to see it written, they do not want to figure it out themselves...

Reply:
My question is how large is the overlap between the groups: Group 1 can replace unix path by Windows path easily. Group 2 is not distracted or confused by the `#` comments. I'm guessing pretty small. In other words, if the reader is familiar with unix/Bash syntax, the Windows piece is not needed. And if a unix path is a stumbling block, so is a Bash comment.

Reply:
in my experience, many manual examples are too *nix biased and exclude _average_ windows user by not giving literal working windows examples. I agree here with @veroandreo.

Reply:
> ...many manual examples are too *nix biased...

I think what would help is saying that it is a Bash or whatever example so that it would be clear that commands or syntax may be different (on Windows) and that it is not "standard GRASS" but rather "standard Bash/unix/...".

Otherwise, I think we need Python examples in manual pages more than Windows examples anyway.

> ...exclude average windows user...

Similarly to the "dropping terminal" PR, I continue being puzzled about who are these _average Windows users_ attached to their command lines. Isn't that a stereotype about Linux users, not Windows ones? :-)